itou/templates/users/details.html

```diff
@@ -150,12 +147,44 @@ <h2>Données administratives</h2>
                                             <i class="text-disabled">Non renseigné</i>
                                         {% endif %}
                                     </li>
-
+                                    {% if profile.advisor_information %}
+                                        <li>
+                                            <div class="c-info c-info--secondary" id="advisor-info-details-collapsable">
+                                                <button class="c-info__summary collapsed"
+                                                        data-bs-toggle="collapse"
+                                                        data-bs-target="#collpaseAdvisorInfoDetails"
+                                                        aria-expanded="false"
+                                                        aria-controls="collpaseAdvisorInfoDetails"
+                                                        {% matomo_event "gps" "clic-coordonees-conseiller" render_advisor_matomo_option|default:"ailleurs" %}>
```

Comment:
Le nom de l’action est `clic` dans les autres évènements GPS.
```suggestion
                                                        {% matomo_event "gps" "clic" "coordonnees-conseiller-{{ render_advisor_matomo_option|default:"ailleurs }}" %}>
```